src/test/java/RestAssuredTests/TestPost.java

```diff
@@ -3,47 +3,40 @@
 import APICalls.PostCall;
 import io.restassured.RestAssured;
 import io.restassured.response.Response;
-import io.restassured.specification.RequestSpecification;
-import org.testng.Assert;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 import utility.PropertyManager;
 
-import java.io.FileWriter;
 import java.io.IOException;
-import java.util.Properties;
 
 import static org.hamcrest.Matchers.equalTo;
-import static utility.PropertyManager.getProperties;
+import static utility.PropertyManager.setNewId;
 
 public class TestPost extends PostCall {
+
+    Response response = RestAssured.given().when().header("Content-Type", "application/json").post(POST_URL);
```

Comment:
instead of making the API call in the constructor, move it to a @BeforeClass method, this will ensure the board is created once per test class, not per instance of the class